change/@ni-jasmine-parameterized-7779e285-bc01-469c-8e6d-ae0b039d8273.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Make this a minor for the moment, let's get one release out and tested after publish and then bump to major

Reply:
```suggestion
  "type": "minor",
```